pkg/oomstore/feature_test.go

```diff
@@ -78,12 +75,9 @@ func TestCreateBatchFeature(t *testing.T) {
 				}
 				metadataStore.EXPECT().CreateFeature(ctx, metadataOpt).Return(0, nil)
 			}
-			_, err := store.CreateBatchFeature(ctx, tc.opt)
-			if tc.expectError {
-				assert.Error(t, err, fmt.Errorf("expected batch feature group, got %s feature group", tc.group.Category))
-			} else {
-				assert.NoError(t, err)
-			}
+
+			_, err := store.CreateFeature(ctx, tc.opt)
+			assert.NoError(t, err)
```

Comment:
add test case for non-batch/stream feature?

Reply:
Updated unit-test. whether the current feature is a batch or a stream is determined by the group. So here we can only determine if the group does not exist